walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/VerifiedIdManifestIssuerStyle.kt

```diff
@@ -10,11 +10,5 @@ package com.microsoft.walletlibrary.requests.styles
  */
 data class VerifiedIdManifestIssuerStyle(
     // Name of the requester.
-    override val requester: String,
-
-    // Locale of the requester.
-    val locale: String,
-
-    // Logo of the requester. If a url is present in contract, it is fetched to this.
-    val logo: Logo? = null
+    override val requester: String
```

Comment:
I have "name" as this property now in iOS. What do you think should be the property name here? I am open to either.

Reply:
yeah name is good. I didn't notice the difference. I will change it.